client/src/modules/templates/breadcrumb.tmpl.html

```diff
@@ -18,9 +18,9 @@
             <i class="fa" ng-class="'{{d.icon}}'"></i> {{ d.selected | translate }} <span class="caret"></span>
           </a>
           <ul class="pull-right" uib-dropdown-menu>
-            <li ng-repeat="opt in d.option">
-              <a href ng-click="vm.helperDropdown(opt, d)" id="{{ vm.prefix + opt.id }}" data-method="{{ opt.dataMethod }}" translate>
-                {{ opt.label }}
+            <li ng-repeat="opt in d.option"  class="pull-left">
```

Comment:
Just for Aligment of icon in DropDonws
![diffpic](https://cloud.githubusercontent.com/assets/8035588/25703209/e531894e-30d4-11e7-851c-2c4bc8079565.png)